vignettes/DataAnalysis.Rmd

```diff
@@ -32,17 +32,17 @@ dt <- function(data){
 
 # Introduction
 
-This vignette walks users through the mechanics of the `Assess()` functions within the `{gsm}` package. `{gsm}` leverages Key Risk Indicators (KRIs) and thresholds to conduct study-level and site-level Risk Based Monitoring for clinical trials.
+This vignette walks users through the mechanics of the functions that produce all of the Analytics output within the `{gsm}` package. `{gsm}` leverages Key Risk Indicators (KRIs) and thresholds to conduct study-level and site-level Risk Based Monitoring for clinical trials.
 
-The `Assess()` functions provide data frames, visualizations, and metadata to be used in reporting and error checking at clinical sites. The image below illustrates the supporting functions that feed into the workflow of the `Assess()` functions.
+These functions provide data frames, visualizations, and metadata to be used in reporting and error checking at clinical sites. The image below illustrates the supporting functions that feed into the `yaml` workflow that is specified in each analysis workflow.
 
 ![](data-analysis.PNG){width="100%"}
```

Comment:
made an edit to the image, but its still tough to read. i might consider increasing the maxwidth of all of the vignettes to accommodate the images we want to include, but i'll make that a separate issue. thanks for the note!

Reply:
I won't block the approval for it, but maybe make the images vertical in a future PR? I know they take up a lot of space that way, but it might be (much) easier to read.